getting_started/scripting/gdscript/gdscript_basics.rst

```diff
@@ -323,26 +325,28 @@ considered a comment.
     # This is a comment.
 
 
-Multi-line comments can be created using """ (three quotes in a row) at
+Multi-line comments can be created using ``"""`` (three quotes in a row) at
 the beginning and end of a block of text. Note that this creates a string,
 therefore, it will not be stripped away when the script is compiled.
 
-    ::
+::
 
-        """ Everything on these
-        lines is considered
-        a comment. """
+    """
+    Everything on these
+    lines is considered
+    a comment.
+    """
```

Comment:
Technically, these are not treated as comments, but as string literals.

Reply:
Should I remove this paragraph entirely? Since Godot 3.1, multiline strings are no longer highlighted as comments. Besides, this causes a string to be allocated every time the code is run, which may cause performance issues in tight loops.

Reply:
That might be a better approach, rather than encouraging its use.